.github/workflows/soundness.yml

```diff
@@ -177,3 +185,21 @@ jobs:
         run: |
           apt-get -qq update && apt-get -qq -y install shellcheck
           git ls-files -z '*.sh' | xargs -0 shellcheck
+  python-lint:
+    name: Python lint
+    if: ${{ inputs.python_lint_enabled }}
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ inputs.python_lint_container_image }}
```

Comment:
Similar comment as in your other PR I don't think we need a container here. The base runner comes with python and pip installed.

Reply:
Nice. Good suggestion. I updated it.